lib/post/pages/post_page.dart

```diff
@@ -319,6 +334,11 @@ class _PostPageState extends State<PostPage> {
         ),
       ),
     );
+
+    // Can't add a listener until the primaryScrollController is build by the widget
+    _itemScrollController.primaryScrollController?.addListener(_onScroll);
```

Comment:
I think instead of doing this, we can run this in the initState to add the listener?

```dart
WidgetsBinding.instance.addPostFrameCallback((_) => _itemScrollController.primaryScrollController?.addListener(_onScroll););
```

Reply:
This works in `post_page_succes` but not here. 😔 I committed the change there. Let me know if there's anything else you want me to try.

Reply:
Thanks to @ajsosa's change in #554, we don't need a listener in `post_page` anymore, so I think your suggestion is resolved!